effects/cargo/test/default.nix

```diff
@@ -0,0 +1,8 @@
+{ pkgs }:
+let
+  effects = import ../../default.nix effects pkgs;
+  inherit (effects) cargoPublish;
+in
+cargoPublish {
+  secretName = "cargo-api-token";
+}
```

Comment:
This looks a bit incomplete without an actual crate to publish.

thoughts:
I didn't find a packaged crate registry in the NixOS search, making offline testing non-trivial.
Maybe I didn't know what to look for though.
- `cargo-local-registry` seems to be more like a proxy
- `alexandrie` not packaged
- `crates.io` not packaged (and perhaps not geared towards self hosting)
- No NixOS option search results for `crate`

Does `crates.io` have a dry-run mode or something? Otherwise I guess we'll just publish a tiny dummy package there.

Reply:
There's a `--dry-run` option, that could work.